figures/arch.dot

```diff
@@ -14,6 +14,7 @@ digraph G {
 
         Jupyter [label="Jupyter Notebook server +\n SQLFlow magic command"];
         SQLFlow [label="SQLFlow server"];
+        ModelZoo [label="Model Zoo server\n"];
```

Comment:
It seems not enough to add only the model zoo node here -- the model zoo needs a Docker registry, file storage, and database system, right?

Reply:
That's true, yet we do not need to deploy a Docker registry or distributed file system for model storage, we use the publicly available service for the model zoo. I'll add them out side the VM.